### ISSUE TITLE
I have a question about "wslay_event_queue_fragmented_msg"

### ISSUE THREAD

**Issue by rock59** [tatsuhiro-t/wslay#65]
Does "wslay_event_queue_fragmented_msg" copy messages just like "wslay_event_queue_msg" ?

**Comment by ljluestc** [comment]
Yes, the function wslay_event_queue_fragmented_msg in the Wslay library behaves similarly to wslay_event_queue_msg, but it is specifically designed for sending fragmented WebSocket messages. Just like wslay_event_queue_msg, it allows you to queue a message to be sent, but it also provides the ability to send large messages in multiple fragments.

WebSocket messages can be fragmented when their payload is too large to fit into a single frame. In such cases, the message is divided into smaller fragments, each sent in its own WebSocket frame. The receiving end then needs to reassemble these fragments to reconstruct the original message.

Here's a high-level comparison between the two functions:

wslay_event_queue_msg:

Used for sending complete WebSocket messages.
The entire message is provided as a single payload.
The library handles creating and sending the appropriate WebSocket frames.
wslay_event_queue_fragmented_msg:

Used for sending large WebSocket messages that need to be fragmented.
Allows you to send a message in multiple fragments (frames).
You provide each fragment of the message along with a flag indicating whether it's the beginning, middle, or end of the fragmented message.
Useful for scenarios where you need to send large data over WebSocket connections while respecting frame size limits.
Both functions serve different purposes, and you should choose the appropriate one based on your use case. If you're dealing with messages that fit within a single frame, you can use wslay_event_queue_msg. If your messages are large and need to be fragmented, use wslay_event_queue_fragmented_msg to send the message in parts.

**Comment by rock59** [comment]
@ljluestc Thanks
